index.js

```diff
@@ -12,7 +12,7 @@ function parse (args, opts) {
   args = tokenizeArgString(args)
   // aliases might have transitive relationships, normalize this.
   var aliases = combineAliases(opts.alias || {})
-  var configuration = loadConfiguration(opts.cwd || path.resolve(path.dirname(__filename), '../'))
+  var configuration = loadConfiguration(opts.cwd || rootPath.path)
```

Comment:
(it looks like) you don't need to even use app-root-path because pkg-conf uses find-root and find-root uses some of the same logic app-root-path uses: https://github.com/jden/find-root/blob/master/index.js#L5